changelog.txt

```diff
@@ -1,5 +1,10 @@
 *** Changelog ***
 
+2021.03.01 - version 2.1.0
```

Comment:
I think this will probably need to be changed to 2021.03.02 now.

Reply:
https://github.com/woocommerce/sensei-certificates/pull/259/commits/d18da19e7e48d768889da32d348e34095259558f ✅